<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="scrcpy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scrcpy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse APK</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listExported">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Exported Activities :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adbLaunch">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addExtras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Extras&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="extrasTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::AllEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="removeExtras">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure Intent :</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>210</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>20</width>
      <height>651</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="actionLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;E.g. &lt;span style=&quot; font-family:'Helvetica'; font-size:12px;&quot;&gt;android.intent.action.VIEW&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dataLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Value of -d&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Action :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="activityText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;E.g. &lt;span style=&quot; font-family:'Helvetica'; font-size:12px;&quot;&gt;android.intent.action.VIEW&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>15</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Java Code :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apkCodeGenerate">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure Extras :</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Using ADB :</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Using APK :</string>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>396</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="javaCode">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>581</width>
      <height>621</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::InputMethodHint::ImhMultiLine|Qt::InputMethodHint::ImhNoAutoUppercase|Qt::InputMethodHint::ImhNoEditMenu|Qt::InputMethodHint::ImhNoPredictiveText</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Menlo'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="jadx">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jadx-gui</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
